pkg/stacks/unpack.go

```diff
@@ -311,21 +314,25 @@ func (sp *StackPackage) applyRules() error {
 
 	// owned CRD rules
 	orderedKeys := orderStackCRDKeys(sp.CRDs)
-
 	for _, k := range orderedKeys {
 		crd := sp.CRDs[k]
-		// TODO(displague) deal with Versions (multiple per crd)
-		gv := schema.GroupVersion{Group: crd.Spec.Group, Version: crd.Spec.Version}
-		rule := generateRBAC(crd.Spec.Names.Plural, gv.String())
+		rule := generateRBAC(crd.Spec.Names.Plural, crd.Spec.Group)
 		rbac.Rules = append(rbac.Rules, rule)
 	}
 
 	// dependency based rules
 	for _, dependency := range sp.Stack.Spec.DependsOn {
-		if dependency.CustomResourceDefinition != "" {
-			_, gk := schema.ParseKindArg(dependency.CustomResourceDefinition)
+		crd := dependency.CustomResourceDefinition
+		if crd != "" {
+			// versions are not allowed in RBAC PolicyRules, remove any trailing version denoted by a "/"
+			// e.g., kind.group.com/v1alpha1 -> kind.group.com
+			if i := strings.Index(crd, "/"); i != -1 {
+				crd = crd[:i]
+			}
+
+			gk := schema.ParseGroupKind(crd)
 			if gk.Group == "" || gk.Kind == "" {
-				return errors.New(fmt.Sprintf("cannot parse CustomResourceDefinition %q as Kind and GroupVersion", dependency.CustomResourceDefinition))
+				return errors.New(fmt.Sprintf("cannot parse CustomResourceDefinition %q as Kind and Group", crd))
```

Comment:
Nitpick: in an error condition, it may be helpful for the human to have the full original input too, before it was trimmed. Did you already consider including the original `dependency.CustomResourceDefinition` here as well?

Reply:
i thought about that, but I wanted to keep the error message focused exactly what was passed (and failed) to the ParseGroupKind function, so as to not introduce any ambiguity.